config/routes.rb

```diff
@@ -26,6 +26,7 @@ def self.matches?(request)
   else
     get("/auth/dfe/callback" => "sessions#callback")
     get("/auth/dfe/sign-out" => "sessions#destroy", as: :sign_out)
+    get "/auth/failure" => "sessions#failure"
```

Comment:
Nitpick, but can we be consistent with using `to:`.

```suggestion
    get "/auth/failure", to: "sessions#failure"
```